Automattic/wp-calypso
# Fixes a couple of styling bugs in the Advertising redesign page

This PR fixes some issues in the new Advertising page. 

## Proposed Changes

The issues were: 
* Inconsistency between All and Everywhere on the Campaign details page
In some cases the backend is returning blank in the countries field. In those cases, I changed to use `Everywhere` to align with the correct value.

* The first API fetch for posts is missing the default order parameter 
I fixed the default filter when the page load

* Improve alignment for stats columns on the tables (campaigns & posts)
To help the narrow vews, I am changing the alignment of the stats in the tables to be centered. This way, the columns won't stick together if you have 0 data.

* Fix missing border on Order by filter for posts in mobile
I added the filter for mobile views

## Testing Instructions

* Open the Live preview and navigate to the Advertising page (Tools->Advertising)
* Check in the Network tab, and verify that the first call to `/blaze/posts` endpoint is sending the correct filter
* Shrink your page to a mobile viewport, and click on the order by filter dropdown
* Verify that the dropdown box no has a border top.

For the campaigns details fix you will need to have a completed campaign that the backend returns blank in the countries field. (there is a bug in the backend that we are working on). If you don't have any, you can hardcode the value [here](https://github.com/Automattic/wp-calypso/blob/48f924bd31dd33362a390c711235d6925cc92de8/client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx#L110).

Then go to the campaign list page and verify that the default value for the countries field is `Everywhere` 
![Screenshot 2023-06-13 at 19 23 27](https://github.com/Automattic/wp-calypso/assets/1258162/d2d6be16-7c68-443d-93a2-d7ec01895bc4)

## Pre-merge Checklist
